docs(Fx142): css ampersand in at-scope content and `:scope` (#40814)
mdn/content: files/en-us/web/css/@scope/index.md

### Specificity in `@scope`

> Inside an `@scope` rule, both bare selectors and `&` are treated as if `:where(:scope)` were prepended, meaning they select the scope root with zero specificity.
> Because [`:where()`](/en-US/docs/Web/CSS/:where) has zero specificity, the only specificity comes from `img` (`0-0-1`).

Reviewer: something like this...to chain the first sentence with `:where()`'s zero specificity:
```suggestion
Inside an `@scope` rule, both bare selectors and leading `&` behave as if `:where(:scope)` were prepended to the selector. Because [`:where()`](/en-US/docs/Web/CSS/:where) has zero specificity, bare selectors and `&` add zero weight and only the specificity of the rest of the selector counts.

In both cases in the following example, the only specificity comes from `img` (`0-0-1`).
```

- Might be important to say "leading `&`" or "`&` at the start of a selector" - to differentiate it from the bare `&`.
WDYT?
IIUC `&` by itself will target the scope root. (in contrast, `& img` will target elements within the scope)

- We could also add to second para: "`& img` is equivalent to writing `:where(:scope) img`."

```css
@scope (.article-body) {
  /* img has a specificity of 0-0-1, as expected */
  img {
    /* … */
  }

  /* & img also has a specificity of 0-0-1 */
  & img {
    /* … */
  }
}
```

> By contrast, using `:scope` explicitly selects the scope root with class specificity.
> Like other pseudo-classes, `:scope` has a specificity of `0-1-0`.
> In the following example, `:scope img` has a specificity of `0-1-1`:

Reviewer: could probably tie up the first two sentences
```suggestion
By contrast, using `:scope` explicitly selects the scope root and adds class-level specificity (`0-1-0`), since `:scope` is a pseudo-class.
In the following example, `:scope img` has a specificity of `0-1-1`:
```
